aiohttp/streams.py

```diff
@@ -388,31 +388,29 @@ def end_http_chunk_receiving(self) -> None:
         of the data corresponds to the end of a HTTP chunk , otherwise it is
         always False.
```

Comment:
@asvetlov I would change the API. This function is meaningless when chunked encoding is not used. So I would raise exception if this function used on non-chunked encoding (can be tested using `self._http_chunk_splits is None`)